Rakefile

```diff
@@ -508,8 +508,8 @@ namespace :kokoro do
   end
 
   task :publish_docs do
-    # kokoro.devsite
-    kokoro.cloudrad
+    kokoro.devsite
+    # kokoro.cloudrad
```

Comment:
This code from the rakefile and rakelib is actually dead now. The logic was moved into `.toys/kokoro/.lib/releaser.rb`.